src/app/components/organisms/EditableInformation/EditableInformation.stories.tsx

```diff
@@ -33,8 +33,12 @@ const BaseComponent = <T extends DropdownSelectOption, U extends DropdownMultiSe
     const [saveWarnings, setSaveWarnings] = useState<Array<string>>(undefined as unknown as string[]);
 
     useEffect(() => {
-        setIsEditing(saveErrors && saveErrors.length !== 0);
-    }, [saveErrors]);
+        setIsEditing(isEditingMode);
+    }, [isEditingMode]);
+
+    useEffect(() => {
+        setIsEditing((saveErrors && saveErrors.length !== 0) || isEditingMode);
+    }, [isEditingMode, saveErrors]);
```

Comment:
You don't need the first useEffect on isEditingMode because the secode useEffect will also fire every change of isEditingMode and will do setIsEditing(isEditingMode); when saveErrors is empty.